### PR TITLE
Add info about accessing container images from private registries

### PR DESCRIPTION
**Pull request by dmitsh**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Add info about accessing container images from private registries

## Issue reference

https://github.com/dapr/dapr/issues/3298

### CODE REVIEW

**Comment by dmitsh** [comment]
@AaronCrawfis , @msfussell , please review style/wording.

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ Deploying and running a Dapr enabled application into your Kubernetes cluster is
     dapr.io/config: "tracing"
 ```
 
+#### Container images from private registries
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md]:
```suggestion
## Pull container images from private registries

```

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md [hunk 2]
```diff
@@ -32,6 +32,10 @@ Deploying and running a Dapr enabled application into your Kubernetes cluster is
...
+Dapr works seamlessly with any container image, regardless of its origin. For information about pulling images from a private registry read [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). If you are using Azure Container Registry with Azure Kubernetes Service, check [AKS documentation](https://docs.microsoft.com/en-us/azure/aks/cluster-container-registry-integration).
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md]:
```suggestion
Dapr works seamlessly with any container image, regardless of its origin, if the cluster is configured to pull from the registry. For information about pulling images from a private registry, reference the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). If you are using Azure Container Registry with Azure Kubernetes Service, reference the [AKS documentation](https://docs.microsoft.com/en-us/azure/aks/cluster-container-registry-integration).
```

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md [hunk 3]
```diff
@@ -32,6 +32,10 @@ Deploying and running a Dapr enabled application into your Kubernetes cluster is
...
+
+
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md:43]:
```suggestion
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md [hunk 4]
```diff
@@ -32,6 +32,11 @@ Deploying and running a Dapr enabled application into your Kubernetes cluster is
     dapr.io/config: "tracing"
 ```
 
+## Pull container images from private registries
+
+Dapr works seamlessly with any container image, regardless of its origin, if the cluster is configured to pull from the registry. For information about pulling images from a private registry, reference the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). If you are using Azure Container Registry with Azure Kubernetes Service, reference the [AKS documentation](https://docs.microsoft.com/en-us/azure/aks/cluster-container-registry-integration).
```
Comment (AaronCrawfis) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md]:
```suggestion
Dapr works seamlessly with any user application container image, regardless of its origin. Simply init Dapr and add the [Dapr annotations]({{< ref kubernetes-annotations >}}) to your Kubernetes definition to add the Dapr sidecar.

The Dapr control-plane and sidecar images come from the [daprio Docker Hub](https://hub.docker.com/u/daprio) container registry.

For information about pulling your application images from a private registry, reference the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). If you are using Azure Container Registry with Azure Kubernetes Service, reference the [AKS documentation](https://docs.microsoft.com/en-us/azure/aks/cluster-container-registry-integration).
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Clarified after revisiting context

**Review by AaronCrawfis — APPROVED** [review]

**Comment by orizohar** [comment]
@dmitsh I see this is opened for 1.3 but I see no reason why it couldn't have been PRd into 1.2 right? As we are pretty close to releasing 1.3 it doesn't matter much but in general better to push these into the current branch

**Pull request merged**